### ISSUE TITLE
Contains should get the inner most match

### ISSUE THREAD

**Issue by NivedhaSenthil** [getgauge/taiko#258]
### Expected behavior
```
(async () => {
    try {
        await openBrowser();
        await goto("gauge.org");
        await click("Get Started");
        await highlight(contains("Quick Install"));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
Above script should highlight the `Quick Install ` link. 

### Actual behavior

It highlights the whole menu section like below,
<img width="1198" alt="screen shot 2018-11-13 at 10 55 28 am" src="https://user-images.githubusercontent.com/6358540/48392806-c8520680-e732-11e8-805d-b4aee2c645b4.png">

**Comment by NivedhaSenthil** [comment]
Fix available in d4c1a45

**Comment by sguptatw** [comment]
**Question**
Here the script says `highlight(contains(....))` and not `highlight(link(....))`
When `highlight(link("Quick Install"))` is used it highlights only the given portion. Why is it expected to highlight only "Quick Install" when contains is used?
